### PR TITLE
Added links for Release Notes to API services that have them.

### PR DESCRIPTION
**Pull request by meker12**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -356,6 +367,7 @@
                                    <ul>
                                        <li><a href="/docs/cloud-monitoring/getting-started/">Quick Start</a></li>
                                        <li><a href="/docs/cloud-monitoring/v1/developer-guide/#document-api-reference">API Reference</a></li>
+                                       <li><a href="/docs/cloud-monitoring/v1/developer-guide//developer-guide/#release-notes">Release Notes</a></li>
```
Comment (smashwilson) [inline index.rst]: > `v1/developer-guide//developer-guide/`

:point_up: Probably unintentional?

Comment (meker12) [inline index.rst]: Yes!  Thanks for catching that.

File: index.rst [hunk 2]
```diff
@@ -356,6 +367,7 @@
                                    <ul>
                                        <li><a href="/docs/cloud-monitoring/getting-started/">Quick Start</a></li>
                                        <li><a href="/docs/cloud-monitoring/v1/developer-guide/#document-api-reference">API Reference</a></li>
+                                       <li><a href="/docs/cloud-monitoring/v1/developer-guide/developer-guide/#release-notes">Release Notes</a></li>
```
Comment (smashwilson) [inline index.rst]: Um, it's still there?

Comment (meker12) [inline index.rst]: I didn't realize the double "developer-guide".  I removed the "//".  Thanks for catching this, and sorry I didn't notice yesterday.

**Pull request merged**